### PR TITLE
Navigate cleanup

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Removes the first argument to navigate (which was not used), and changes the now second to be an object which will make changing it in the future be easier

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts [hunk 1]
```diff
@@ -100,11 +100,9 @@ export class ZHANetworkVisualizationPage extends LitElement {
       if (ieee) {
         const device = this._devices.get(ieee);
         if (device) {
-          navigate(
-            this,
-            `/config/devices/device/${device.device_reg_id}`,
-            false
-          );
+          navigate(`/config/devices/device/${device.device_reg_id}`, {
+            replace: true,
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts:104]: This was false?

Comment (ludeeus) [reply src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts:104]: oh... my bad, there might be others as well with that, will do a scan and adjust

Comment (ludeeus) [reply src/panels/config/integrations/integration-panels/zha/zha-network-visualization-page.ts]: Looks like that was the only one

**Review by bramkragten — APPROVED** [review]

File: hassio/src/ingress-view/hassio-ingress-view.ts [hunk 2]
```diff
@@ -157,8 +157,8 @@ class HassioIngressView extends LitElement {
       await showAlertDialog(this, {
         text: "Add-on is not running. Please start it first",
         title: addon.name,
+        confirm: () => navigate(`/hassio/addon/${addon.slug}`),
```
Comment (bramkragten) [inline hassio/src/ingress-view/hassio-ingress-view.ts:160]: No replace anymore?

**Pull request merged**